dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -425,8 +424,8 @@ export const Explorer = ({
   const prepareAvailability = async () => {
     setSelectedContentTab(TAB_CHART_ID);
     setTriggerAvailability(true);
-    await setTempQuery(DEFAULT_AVAILABILITY_QUERY);
```

Comment:
This removes the crucial functionality of the Set Availability buttons. Will this be added somewhere else??

Reply:
@eugenesk24 , @mengweieric 

Can you please let us know the functionality of set availability button since we are unaware of it. We had assumed that this is just a default query that fetches data on the initial load of the newly created app. Let us know if we are not on the same page.

Reply:
We have added back DEFAULT_AVAILABILITY_QUERY as it was present initially